dora/users/views.py

```diff
@@ -13,3 +16,12 @@ class Meta:
             "newsletter",
         ]
         read_only_fields = ["email"]
+
+
+@sensitive_post_parameters(["main_activity"])
+@api_view(["PATCH"])
+@permission_classes([permissions.IsAuthenticated])
+def update_main_activity(request):
+    setattr(request.user, "main_activity", request.data.get("main_activity"))
+    request.user.save()
+    return Response("ok", status=200)
```

Comment:
```suggestion
    return Response(status=204)
```
Non ça ne rime à rien de renvoyer "ok". Soit tu renvoies l'objet modifié, avec un code 200, soit tu ne renvoies rien, avec un code 204